src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -152,7 +154,12 @@ def save
         begin
           authorize @package, :update?
 
-          Package.verify_file!(@package, '_patchinfo', xml)
+          begin
+            Package.verify_file!(@package, '_patchinfo', xml)
+          rescue APIException => e
```

Comment:
was it me who wrote this code or made a webui migration without the right error handling?
Don't complain that I fix after you :/